### PR TITLE
Add liveness/readiness to security faq

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

File: _docs/tasks/security/faq.md [hunk 1]
```diff
@@ -43,9 +43,22 @@ type: markdown
...
+  for health check, e.g., one can install curl in the service pod and curl itself
+  within the pod. The Istio team is actively working on a solution.
+
+  An example of readinessProbe:
+
+  ```
+  livenessProbe:
+    exec:
+      command:
+      - /bin/sh
```
Comment (ldemailly) [inline _docs/tasks/security/faq.md]: don't do the /bin/sh it's unecessary for one command

File: _docs/tasks/security/faq.md [hunk 2]
```diff
@@ -43,9 +43,22 @@ type: markdown
...
+      - -c
+      - curl -f http://localhost:{$PORT}
```
Comment (ldemailly) [inline _docs/tasks/security/faq.md]: add /health to the url for example

File: _docs/tasks/security/faq.md [hunk 3]
```diff
@@ -43,9 +43,22 @@ type: markdown
 
   If Istio Auth is enabled, http and tcp health check from kubelet will not
   work since they do not have Istio Auth issued certs. A workaround is to
-  use command option for health check, e.g., one can install curl in the
-  service pod and curl itself within the pod. The Istio team is actively
-  working on a solution.
+  use [command option](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-a-liveness-command)
```
Comment (ldemailly) [inline _docs/tasks/security/faq.md]: use a [liveness command](...)

Comment (wattli) [reply _docs/tasks/security/faq.md]: done

File: _docs/tasks/security/faq.md (hunk 1 above)
Comment (wattli) [reply _docs/tasks/security/faq.md]: done

File: _docs/tasks/security/faq.md (hunk 2 above)
Comment (wattli) [reply _docs/tasks/security/faq.md]: in reality, many people just simply check localhost:port, i.e., they don't use /health for health check. I feel the current way will be easier to understand (for those who use /health, they will simply add /health, but if we go the other way, i.e., add /health to the url, for those who do not use /health, they will not sure what to do).

File: _docs/tasks/security/faq.md [hunk 4]
```diff
@@ -43,9 +43,22 @@ type: markdown
...
+
+  An example of readinessProbe:
+
+  ```
+  livenessProbe:
+    exec:
+      command:
+      - curl
+      - -f
+      - http://localhost:{$PORT}
```
Comment (ldemailly) [inline _docs/tasks/security/faq.md]: the {$PORT} is strange (that’s meant as an example/ for user to change to the actual port right?)
what about : 
```
   - http://localhost:8080/health # replace with your actual port and health endpoint
```

**Pull request merged**